overcast_to_sqlite/html/page.py

```
@@ -26,6 +27,57 @@
 
+class HTMLTagFixer(HTMLParser):
+    def __init__(self):
+        super().__init__()
+        self.open_tags = []
+        self.output = []
+        
+    def handle_starttag(self, tag, attrs):
+        # Self-closing tags don't need closing tags
+        self_closing_tags = {'br', 'hr', 'img', 'input', 'meta', 'link', 'area', 'base', 'col', 'embed', 'source', 'track', 'wbr'}
+        
+        attr_str = ''.join(f' {name}="{value}"' for name, value in attrs)
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)